fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/ThreadLocalContextUtil.java

```diff
@@ -26,29 +31,25 @@
  */
 public final class ThreadLocalContextUtil {
 
-    private ThreadLocalContextUtil() {
-
-    }
-
     public static final String CONTEXT_TENANTS = "tenants";
-
     private static final ThreadLocal<String> contextHolder = new ThreadLocal<>();
-
-    private static final ThreadLocal<FineractPlatformTenant> tenantcontext = new ThreadLocal<>();
-
+    private static final ThreadLocal<FineractPlatformTenant> tenantContext = new ThreadLocal<>();
     private static final ThreadLocal<String> authTokenContext = new ThreadLocal<>();
+    private static final ThreadLocal<HashMap<BusinessDateType, LocalDate>> businessDateContext = new ThreadLocal<>();
```

Comment:
Do you think we could switch the `HashMap` into a regular `Map` interface? I'd say let's try to be as independent from a specific implementation as possible.